Source/WebKit/UIProcess/ProcessThrottler.cpp

```diff
@@ -289,6 +312,11 @@ void ProcessThrottler::setAllowsActivities(bool allow)
         invalidateAllActivities();
 }
 
+void ProcessThrottler::enableFullSuspension()
+{
+    m_fullySuspend = true;
```

Comment:
Per WebKit coding style, boolean variables need a prefix. Here probably `m_shouldFullySuspend`.

Reply:
Thanks, fixed

Reply:
But I am still confused by the "fully suspend" naming :)

Reply:
🤦‍♂️thanks fixed now